packages/phone-number-privacy/src/index.ts

```diff
@@ -1,21 +1,148 @@
+import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
 import { BLINDBLS } from 'bls12377js-blind'
 import * as functions from 'firebase-functions'
+import fetch from 'node-fetch'
 import config, { connectToDatabase } from './config'
+import { ACCOUNTS_COLUMNS, ACCOUNTS_TABLE } from './models/account'
+import { getContractKit } from './web3/contracts'
+
+export const NUM_ATTESTATIONS_REQUIRED = 3
+
+const contractKit = getContractKit()
+const knex = connectToDatabase()
 
 export const getSalt = functions.https.onRequest((request, response) => {
-  const privateKey = new Buffer(config.salt.key)
+  confirmUser()
+  Promise.resolve(getRemainingQueryCount(request.body.account, request.body.phoneNumber)).then(
```

Comment:
We should perform thorough validation of any content that's passed to us from an untrusted client.

Reply:
agreed